test-app/src/androidTest/java/com/avito/android/ui/test/RotationFailuresTest.kt

```diff
@@ -14,6 +15,7 @@ class RotationFailuresTest {
     @Rule @JvmField
     val rule = screenRule<StartForResultActivity>(launchActivity = true)
 
+    @Ignore("need isolation")
```

Comment:
cant get the reason.

Reply:
yep, no real reason here. just observation that this ignore fixes a suite